graphene_django/filter/fields.py

```diff
@@ -89,11 +90,22 @@ def connection_resolver(
         **args
     ):
         filter_kwargs = {k: v for k, v in args.items() if k in filtering_args}
-        qs = filterset_class(
+
+        filterset = filterset_class(
             data=filter_kwargs,
             queryset=default_manager.get_queryset(),
             request=info.context,
-        ).qs
+        )
+
+        if not (filterset.is_bound and filterset.form.is_valid()):
+            exc = {
+                str(key): [str(e.message) for e in error_list]
+                for key, error_list in filterset.form.errors.as_data().items()
+            }
+
+            raise GraphQLError(exc)
```

Comment:
I think we're passing a dictionary here, but [GraphQLError](https://github.com/graphql-python/graphql-core/blob/master/graphql/error/base.py#L26) appears to take a `str`. Should we format this into a string? I'm not sure that relying on the stringifying behaviour of `GraphQLError` is the right thing to do.

One alternative would be to wrap up the `dict -> str` processing in a `GraphQLDjangoFilterError` (better name needed). This would:

a) make the code here simpler
b) make it easier to customise the behaviour in the view the API that lets the Django view intercept errors.